site/Lightbox.tsx

```diff
@@ -136,7 +136,9 @@ export const runLightbox = () => {
         document.body.appendChild(lightboxContainer)
     }
     Array.from(
-        document.querySelectorAll<HTMLImageElement>(".article-content img")
+        document.querySelectorAll<HTMLImageElement>(
+            ".article-content img, .lightbox-image"
```

Comment:
Wanna make `lightbox-image` a constant?

Reply:
Is there a convenient place to put constants like this? I only found the grapher-specific ones in `grapher/core/GrapherConstants.ts`

Reply:
@mathisonian I just put it in this file, where I would most expect it. We can move it around if we later need it across TS projects.